lib/pbbuilder/template.rb

```diff
@@ -0,0 +1,91 @@
+# PbbuilderTemplate is an extension of Pbbuilder to be used as a Rails template
+# It adds support for partials.
+class PbbuilderTemplate < Pbbuilder
+  class << self
+    attr_accessor :template_lookup_options
+  end
+
+  self.template_lookup_options = {handlers: [:pbbuilder]}
```

Comment:
Why not make this a constant?

Reply:
Copied from jbuilder: https://github.com/rails/jbuilder/blob/770abdb6b4f027ef577c16fe1f88bf0ed2db46e4/lib/jbuilder/jbuilder_template.rb#L10

(presumably you could set extra options)